src/server/server.js

```diff
@@ -43,7 +43,7 @@ app.use("/users", routers.userRouter);
 // Index page routing
 app.use("/index", routers.indexRouter);
 
-const port = process.env.PORT;
+const port = process.env.PORT || 5000;
```

Comment:
move the magic number to meaningful const

Reply:
done